core/agent_loop.py

```
@@ -300,14 +515,30 @@ def _react_think(self, task: str) -> Dict:
+        cli = get_cli()
+        
         if action["action"] == "tool_call":
             logger.info(f"Executing tool: {action['tool']}")
             
-            # 参数映射：处理常见的参数名差异
+            # 显示工具调用
             tool_name = action["tool"]
             arguments = action.get("arguments", {}).copy()
             
+            # 检查是否为 HITL 工具
+            if tool_name in HITL_TOOL_NAMES:
```

Ruff (0.15.9)

[error] Undefined name `HITL_TOOL_NAMES`

(F821)